source/.bazelrc

```diff
@@ -21,7 +21,7 @@ build --cxxopt='-D_GLIBCXX_USE_CXX11_ABI=0'
 build --extra_toolchains=@bazel_tools//tools/python:autodetecting_toolchain_nonstrict
 
 # Build with coverage info
-build:coverage --copt='-fprofile-instr-generate' --copt='-fcoverage-mapping' --linkopt='-fprofile-instr-generate' --linkopt='-fcoverage-mapping'
+build:coverage --collect_code_coverage --instrument_test_targets --instrumentation_filter="/java[/:]" --combined_report=lcov --coverage_report_generator=@bazel_tools//tools/test/CoverageOutputGenerator/java/com/google/devtools/coverageoutputgenerator:Main --copt='-fprofile-instr-generate' --copt='-fcoverage-mapping' --linkopt='-fprofile-instr-generate' --linkopt='-fcoverage-mapping'
```

Comment:
Why do we need these flags here and in `build/build.sh`?

Reply:
I thought I reverted it back. It doesn't work as I expected really.